inspirehep/modules/records/api.py

```diff
@@ -586,9 +585,18 @@ def _append_current_control_number(control_number, base_name):
 
         return new_key
 
+    def _get_citations_count(self):
+        """Returns citations count for this record"""
+        count = 0
+        with db.session.begin_nested():
+            count = db.session.query(func.count_citations("%d" % self.id)).first()
```

Comment:
Instead of `first` you can use `one`, that will raise an exception, but in this case I'd perefer the more explanatory exception :)

Reply:
I do not think it is possible to receive more or less than one result here. And still when using `one` it would require to catch exception and raise ours as the default one won't give us any useful information.